keyboards/anavi/knobs3/mcuconf.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
Thank you for the code review. Sorry that I forgot about the license header. I have added it.

Thanks,
Leon